### ISSUE TITLE
Random element not working

### ISSUE THREAD

**Issue by Leovaldez42** [ozrix84/game-of-life#1]
According to the rule _If this condition is true for more than one species on the same element then species type for the new element is chosen randomly_, a random cell should be generated however when I tested it out this does not exactly work. No new cell is formed.

**Comment by ozrix84** [comment]
Yeah, I noticed the same thing when developing it. This piece of software was a job interview assignment, and the instructions the employer supplied me with didn't make much sense in places. They didn't specify the relationship between different cell species either, hence why they don't react to each other in any particular way. These issues are something that will have to be fixed.
